Emby.Server.Implementations/ApplicationHost.cs

```diff
@@ -1561,150 +1409,66 @@ public void Restart()
         /// Gets the composable part assemblies.
         /// </summary>
         /// <returns>IEnumerable{Assembly}.</returns>
-        protected List<Tuple<Assembly, string>> GetComposablePartAssemblies()
+        protected IEnumerable<Assembly> GetComposablePartAssemblies()
         {
-            var list = GetPluginAssemblies(ApplicationPaths.PluginsPath);
+            if (Directory.Exists(ApplicationPaths.PluginsPath))
+            {
+                foreach (var file in Directory.EnumerateFiles(ApplicationPaths.PluginsPath, "*.dll", SearchOption.TopDirectoryOnly))
+                {
+                    Logger.LogInformation("Loading assembly {Path}", file);
+                    yield return Assembly.LoadFile(file);
+                }
+            }
 
             // Gets all plugin assemblies by first reading all bytes of the .dll and calling Assembly.Load against that
```

Comment:
what this comment means and is it still relevant?

Reply:
I think it's no longer relevant as we've switched to `LoadFrom`, right? What about live update of libraries though?